### PR TITLE
Bigquery add adunits and targeting

### PR DESCRIPTION
**Pull request by rori4**
This adds AdUnits and Audiences so Galya can display them in the Data Studio report.

I only have a few questions on this. 

Do all campaigns right now have audience attached to it? If yes then this should be OK

### CODE REVIEW

**Comment by Ivshti** [comment]
@rori4 regarding whether all campaigns have audiences, @IvoPaunov can tell you

**Comment by rori4** [comment]
needs some changes as discussed on slack

File: scripts/bigquery.js [hunk 1]
```diff
@@ -81,6 +108,181 @@ async function createWebsitesTable() {
...
+					mode: 'NULLABLE',
+				},
+			],
+		},
+	})
+	return startImport(
+		AUDIENCES_TABLE_NAME,
+		getMongo()
+			.collection('audiences')
+			.find()
```
Comment (IvoPaunov) [inline scripts/bigquery.js:187]: Cant you add filter `{campaignId: { $exists: true, $ne: null }}` instead `if (!campaign) return // skip if not used in campaign`

**Review by IvoPaunov — CHANGES_REQUESTED** [review]
just fix the audience filter

**Review by IvoPaunov — APPROVED** [review]

**Pull request merged**